lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -419,7 +427,12 @@ struct FunctionCallBuilder: BoundsCheckedThunkBuilder {
       // filter out deleted parameters, i.e. ones where argTypes[i] _contains_ nil
       return type == nil || type! != nil
     }.map { (i: Int, e: FunctionParameterSyntax) in
-      e.with(\.type, (argTypes[i] ?? e.type)!)
+      let param = e.with(\.type, (argTypes[i] ?? e.type)!)
+      let name = param.secondName ?? param.firstName
+      if name.trimmed.text == "_" {
```

Comment:
Why the redundant logic to extract the parameter name and check for `_`?

Reply:
Ah good point!

Reply:
Wait no that redundancy is actually an error:
```
   | `- note: in expansion of macro '_SwiftifyImport' on global function 'ptrNamed(ptr:_:)' here
 6 | func ptrNamed(ptr: UnsafePointer<CInt>, _ len: CInt) {
 7 | }
   +--- macro expansion @_SwiftifyImport -------------------------------
   |1 | @_alwaysEmitIntoClient
   |2 | func ptrNamed(ptr ptr: UnsafeBufferPointer<CInt>) {
   |  |               `- error: extraneous duplicate parameter name; 'ptr' already has an argument label
   |3 |     return unsafe ptrNamed(ptr: ptr.baseAddress!, CInt(exactly: ptr.count)!)
   |4 | }
   +--------------------------------------------------------------------
```
I don't know of any way a C function can import to that syntax, but it doesn't hurt to handle it correctly.